infer/tests/codetoanalyze/java/pulse/taint/NullPointer.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package codetoanalyze.java.pulse;
+
+class NullPointer {
+
+    void source(StringBuilder request) {
+    }
+
+    void foo() {
```

Comment:
Could you rename it to  something like`nullPointerTaintedFalsePositive` to make it more explicit?
Also, it would probably be nice to add a test case for String literals as well.

Reply:
I renamed the methods and added an example for string literals.